File: cs/Iris/SimpleExpressionElement/ConstantExpression.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Iris.SimpleExpressionElement
{
    public class ConstantExpression : SimpleExpression
    {
        public ConstantExpression()
        {

        }

        public override ObjectType getExpressionType()
        {
            return ObjectType.intType;
        }
    }
}

File: java/Iris/src/com/rokuan/iris/expression/ConstantExpression.java
package com.rokuan.iris.expression;

import com.rokuan.iris.types.ObjectType;

public class ConstantExpression extends Expression
{
	public ConstantExpression()
	{

	}

	public ObjectType getExpressionType()
	{
		return ObjectType.intType;
	}
}
